sdks/python/apache_beam/transforms/transforms_keyword_only_args_py3_only_test.py

```diff
@@ -0,0 +1,115 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Unit tests for side inputs."""
+
+from __future__ import absolute_import
+
+import logging
+import sys
+import unittest
+
+import apache_beam as beam  # pylint: disable=unused-import
+from apache_beam.testing.test_pipeline import TestPipeline  # pylint: disable=unused-import
+from apache_beam.testing.util import assert_that  # pylint: disable=unused-import
+from apache_beam.testing.util import equal_to  # pylint: disable=unused-import
+
+IS_PYTHON_2 = sys.version_info[0] == 2
+
+
+# pylint: disable=exec-used
+@unittest.skipIf(IS_PYTHON_2, 'test only in python 3 with kwonly args')
```

Comment:
We came up with a different way of adding Python3-only tests, see https://github.com/apache/beam/pull/9223. Let's follow that pattern here. We would need to rename this file.